.codeclimate.yml

```diff
@@ -56,4 +56,5 @@ exclude_patterns:
 - tools/*
 - examples/*
 - test/*
+- "!bin/*"
```

Comment:
IIRC It's already looking in bin ... It's just not looking at things that are not `.py` codes.

Reply:
Yeah, this was the first (and simplest) test to check that

Reply:
You're right, the documentation [says](https://docs.codeclimate.com/docs/excluding-files-and-folders).

> Furthermore, engines only run their analysis on files with an appropriate file extension for the language being analyzed (.rb for a Ruby engine, .py for a Python engine, etc). Therefore, in most cases you should not need to specify excludes for files in languages other than your target languages

I've asked stackoverflow for any help on this